resources/language/messages.pot

```diff
@@ -2250,5 +2250,5 @@ msgid "English | en"
 msgstr ""
 
 msgctxt "#30702"
-msgid "Exceeded daily allowed subtitles downloads"
+msgid "Reached daily limit for subtitles downloads without login"
```

Comment:
Leftovers

Reply:
fixed.
also fixed lefover in 30703.
thanks.